### DISCUSSION TITLE
404  There isn't a GitHub Pages site here.

### DISCUSSION THREAD

**Discussion by negishubham** [academicpages/academicpages.github.io#2045] [Q&A]
I create my repository by forking the current version. But after my first commit I am getting "404 There isn't a GitHub Pages site here"  error when I try to load the my page. Maybe I am missing something trivial.  

Thanks!

**Comment by rjzupkoii** [comment]
What URL are you using to access the site? It looks like the expected on - https://negishubham.github.io/ - is working.

**Reply by negishubham** [reply]
Yeah, it worked only when I went to "Actions" and was looking at the build time. Not sure if that triggered it. But yes it is working now. Thanks!

**Reply by rjzupkoii** [reply]
GitHub Actions can sometimes take a bit to trigger, so that might have been it. I've been behind on updating things, but that's something I can add to the documentation for the template.
